tests/container_checker/test_container_checker.py

```diff
@@ -203,18 +204,26 @@ def test_container_checker(duthosts, enum_rand_one_per_hwsku_hostname,
     container_name = asic.get_docker_name(service_name)
 
     loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix="container_checker_{}".format(container_name))
-
+    sleep_time = 70
     disabled_containers = get_disabled_container_list(duthost)
 
     skip_containers = disabled_containers[:]
 
     # Skip 'radv' container on devices whose role is not T0/M0.
     if tbinfo["topo"]["type"] not in ["t0", "m0"]:
         skip_containers.append("radv")
-
     pytest_require(service_name not in skip_containers,
                    "Container '{}' is skipped for testing.".format(container_name))
-
+    feature_autorestart_states = duthost.get_container_autorestart_states()
+    if feature_autorestart_states.get(service_name) == 'enabled':
+        disable_container_autorestart(duthost)
+        time.sleep(30)
+    if not is_container_running(duthost, container_name):
+        logger.info("Container '{}' is not running ...".format(container_name))
+        logger.info("Reload config on DuT as Container is not up '{}' ...".format(duthost.hostname))
+        config_reload(duthost, safe_reload=True)
```

Comment:
if container autorestart is disabled (line 219) and that is not saved in the config, then doing a config reload will set it back to what was in the saved config, is that intentional?

Reply:
@SuvarnaMeenakshi 
Config reload will not enable the container auto restart, only a load minigraph will as the config is saved in the function  disable_container_autorestart

![image](https://github.com/sonic-net/sonic-mgmt/assets/135994174/de61c10e-c144-442b-bc0e-239857bcfdfe)